documentation/troubleshooting/modules/ipi-install-troubleshooting-ntp-out-of-sync.adoc

```diff
@@ -99,48 +101,52 @@ spec:
       security:
         tls: {}
       timeouts: {}
+ifeval::[{release} >= 4.6]
+      version: 3.1.0
+endif::[]
+ifeval::[{release} < 4.6]
       version: 2.2.0
+endif::[]
     networkd: {}
     passwd: {}
     storage:
       files:
       - contents:
           source: data:text/plain;charset=utf-8;base64,[text-in-base-64]<1>
-          verification: {}
-        filesystem: root
         group:
           name: root
         mode: 420
+        overwrite: true
```

Comment:
would this need to be removed for version 2.2.0?

Reply:
Yup, added conditional